js/timers.ts

```diff
@@ -181,6 +181,13 @@ function fireTimers(): void {
 
 export type Args = unknown[];
 
+// @internal
```

Comment:
If you don't export it, or it isn't part of any surface of anything else exported it shouldn't leak into the `.d.ts` file.